### PR TITLE
chore: add max width to display example

### PR DESCRIPTION
**Pull request by FRSgit**
add the max with to make display showcase visually nicer

# Checklist

- [x] Self code-reviewed
- [x] Changes documented
- [x] Semantic HTML
- [x] SSR-friendly
- [x] Caching friendly
- [x] a11y for WCAG 2.0 AA
- [x] examples created
- [x] created cypress tests

### CODE REVIEW

File: apps/preview/next/pages/examples/VsfNavigationTop.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ function Example() {
         >
           <div className="border border-primary-400">Column1</div>
           <div className="border border-primary-400">Column2</div>
-          <div className="border border-primary-400">Column3</div>
+          <div className="border border-primary-400">Column3chore/</div>
```
Comment (justyna-13) [inline apps/preview/next/pages/examples/VsfNavigationTop.tsx:72]: is this change also needed?

**Review by justyna-13 — APPROVED** [review]

File: apps/preview/next/pages/examples/VsfNavigationTop.tsx (hunk 1 above)
Comment (FRSgit) [reply apps/preview/next/pages/examples/VsfNavigationTop.tsx:72]: 😄

Comment (FRSgit) [reply apps/preview/next/pages/examples/VsfNavigationTop.tsx]: removed 9890f011278a764cb455421b3b696664356ec92d

**Review by aniamusial — APPROVED** [review]

**Review by AdamPawlinski — APPROVED** [review]
Wdyt about limiting the height of the image in the vertical version? Currently entire component is not visible.

**Pull request merged**